src/sentry/static/sentry/app/views/organizationDashboard/platformList.jsx

```diff
@@ -1,95 +0,0 @@
-import React from 'react';
-import PropTypes from 'prop-types';
-import {withRouter} from 'react-router';
-import styled from 'react-emotion';
-import {Flex} from 'grid-emotion';
-
-import SentryTypes from 'app/proptypes';
-import {t} from 'app/locale';
-import Button from 'app/components/buttons/button';
-import Platformicon from 'app/components/platformicon';
-
-const MAX_PLATFORMS = 5;
-
-class PlatformList extends React.Component {
```

Comment:
Does nothing reference this module?

Reply:
Nope the organizationDashboard was reworked and so I moved it to a shared component